image.go

```diff
@@ -525,6 +526,18 @@ func (c *Client) BuildImage(opts BuildImageOptions) error {
 		}
 	}
 
+	if len(opts.Labels) > 0 {
+		label := make(map[string]string)
+		for k, v := range opts.Labels {
+			label[k] = v
+		}
+		if b, err := json.Marshal(label); err == nil {
+			item := url.Values(map[string][]string{})
+			item.Add("labels", string(b))
+			qs = fmt.Sprintf("%s&%s", qs, item.Encode())
+		}
+	}
+
```

Comment:
The queryString function already does that, so you just need to use ``qs:"labels"`` in the Labels field.